setup.py

```diff
@@ -70,7 +70,7 @@ def build(cls):
         try:
             import sass
         except ModuleNotFoundError:
-            pip.main(['install', 'libsass'])
+            check_call([sys.executable, '-m', 'pip', 'install', 'libsass'])
```

Comment:
@beraldoleal please take a look at #471 when you have the time...
changes are made as requested.
thanks